source/application/AMPIC887DriverServer/AMPIC887EpicsCoordinator.cpp

```diff
@@ -316,9 +346,21 @@ void AMPIC887EpicsCoordinator::onAllConnected(bool connectedState)
 				qDebug() << "Initialization: Setting system velocity setpoint to " << controller_->systemVelocity();
 				systemVelocitySetpoint_->move(controller_->systemVelocity());
 			}
+
+			// Initialize the record rate.
+			if(!recordRateFeedback_->withinTolerance(controller_->recordRate())) {
+				qDebug() << "Initialization: Setting record rate to " << controller_->recordRate();
```

Comment:
All the debugs in the coordinator application are intentional. They are in there so that if someone comes to the server they can see an output of the recent 'events'.